pkg/providers/tinkerbell/tinkerbell.go

```diff
@@ -117,6 +117,8 @@ func NewProvider(
 		}
 	}
 
+	tinkerbellIp, _ := networkutils.GetLocalIP()
```

Comment:
Any reason we aren't checking the error? I know this won't be the final code but we should still check the error.

Reply:
True, NewProvider construct does not return error so I had to ignore it. Should I put this somewhere else?

Reply:
you can get it in the provider factory and inject it from there to the provider